**Description**
Hi Guys, I understand that the official repos don't yet support Ubuntu 20.04 with Py3 implementation, so please feeel free to close if this is expected behaviour. However, when trying to connect a Ubuntu 20.04 to a Ubuntu 18.04 master (both using Salt version 3000.2) I get an "Unexpected error while connecting to salt (master) KeyError: 'os_family'" on the minion.

**Setup**
- Existing salt-master (Ubuntu 18.04, 3000.2).
- New salt-minion (Ubuntu 20.04, 3000.2).
- Minion's key is seen by and accepted by master.


**Steps to Reproduce the behavior**
Attempt to run highstate on the minion after accepting key on master; or simply run `systemctl restart salt-minion.service`; the output of `/var/log/salt/minion` will be flooded with these messages (several per second):

```
2020-04-30 15:49:47,480 [salt.minion      :1058][CRITICAL][17150] Unexpected error while connecting to salt
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/salt/minion.py", line 1037, in _connect_minion
    yield minion.connect_master(failed=failed)
  File "/usr/lib/python3/dist-packages/salt/ext/tornado/gen.py", line 1056, in run
    value = future.result()
  File "/usr/lib/python3/dist-packages/salt/ext/tornado/concurrent.py", line 239, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
  File "/usr/lib/python3/dist-packages/salt/ext/tornado/gen.py", line 1064, in run
    yielded = self.gen.throw(*exc_info)
  File "/usr/lib/python3/dist-packages/salt/minion.py", line 1236, in connect_master
    yield self._post_master_init(master)
  File "/usr/lib/python3/dist-packages/salt/ext/tornado/gen.py", line 1056, in run
    value = future.result()
  File "/usr/lib/python3/dist-packages/salt/ext/tornado/concurrent.py", line 239, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
  File "/usr/lib/python3/dist-packages/salt/ext/tornado/gen.py", line 1070, in run
    yielded = self.gen.send(value)
  File "/usr/lib/python3/dist-packages/salt/minion.py", line 1279, in _post_master_init
    self.schedule = salt.utils.schedule.Schedule(
  File "/usr/lib/python3/dist-packages/salt/utils/schedule.py", line 99, in __new__
    instance.__singleton_init__(opts, functions,
  File "/usr/lib/python3/dist-packages/salt/utils/schedule.py", line 152, in __singleton_init__
    self.time_offset = self.functions.get('timezone.get_offset', lambda: '0000')()
  File "/usr/lib/python3/dist-packages/salt/modules/timezone.py", line 234, in get_offset
    if 'AIX' not in __grains__['os_family']:
  File "/usr/lib/python3/dist-packages/salt/utils/context.py", line 236, in __getitem__
    return self._dict()[key]
KeyError: 'os_family'
```


**Expected behavior**
Connection to the salt-master, allowing highstate to be run.


**Versions Report**
<details><summary>salt --versions-report</summary>
(Provided by running salt --versions-report. Please also mention any differences in master/minion versions.) 

```
salt-minion, running on Ubuntu 20.04 server via the "deb http://repo.saltstack.com/py3/ubuntu/18.04/amd64/latest bionic main" repo.

Salt Version:
           Salt: 3000.2
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.7.3
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
         Jinja2: 2.10.1
        libgit2: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.6.2
   mysql-python: 1.4.4
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.8.2 (default, Mar 13 2020, 10:14:16)
   python-gnupg: 0.4.5
         PyYAML: 5.3.1
          PyZMQ: 18.1.1
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.3.2
 
System Versions:
           dist: ubuntu 20.04 focal
         locale: utf-8
        machine: x86_64
        release: 5.4.0-28-generic
         system: Linux
        version: Ubuntu 20.04 focal
 

salt-master, running on Ubuntu 18.04 server:

Salt Version:
           Salt: 3000.2
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.6.1
      docker-py: Not Installed
          gitdb: 2.0.3
      gitpython: 2.1.8
         Jinja2: 2.10
        libgit2: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.6.9 (default, Apr 18 2020, 01:56:04)
   python-gnupg: 0.4.1
         PyYAML: 3.12
          PyZMQ: 16.0.2
          smmap: 2.0.3
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.2.5
 
System Versions:
           dist: Ubuntu 18.04 bionic
         locale: UTF-8
        machine: x86_64
        release: 4.15.0-96-generic
         system: Linux
        version: Ubuntu 18.04 bionic
```
</details>

